actors/miner/src/lib.rs

```diff
@@ -1031,136 +1031,30 @@ impl Actor {
         updates: Vec<ReplicaUpdateInner>,
     ) -> Result<BitField, ActorError>
     where
-        // + Clone because we messed up and need to keep a copy around between transactions.
-        // https://github.com/filecoin-project/builtin-actors/issues/133
-        RT::Blockstore: Blockstore + Clone,
```

Comment:
How exactly is the Clone requirement going away with this change?

Reply:
I deleted a call to `clone()` previously on line 1057. I am tbh not sure it was necessary in the first place.